Remove mainpage.php

Kamahl19 opened the pull request:
Do not forget to include deletion to `/install_instructions/update_*.xml` file!

Culprit-CZ commented:
What is the reason?

Kamahl19 commented:
The question should be what is the reason for keeping mainpage.php ? We dont use it anymore.

senky commented:
It is unused file. Unused files should be removed.

senky committed:
[i/169] removing visibility: hidden; for .sn-im-smiliesBox

Kamahl19 committed:
[i/76] mainpage.php removed

senky committed:
[i/171] adding line break between "Reason" and "Further information"

senky commented on install_instructions/update_from_0.7.2_to_0.7.3.xml:
```diff
@@ -0,0 +1,64 @@
+			<file name="socialnet/js/jquery.textarea-expander.js"/>
+			<file name="mainpage.php"/>
+		</delete>
+		<php-installer><![CDATA[socialnet/install_sn.php]]></php-installer>
```
Please, include DB update to this PR, too.

Kamahl19 replied:
I dont think so. SQL update is not related with this PR. We will add sql update to the PR before releasing 0.7.3

senky committed:
[i/173] replacing ' . USERS_TABLE . ' with " . USERS_TABLE . "

senky committed:
[i/174] removing float: none; for .sn-fms-user-suggestion

senky committed:
[i/175] hacking collapsing of empty status textarea

senky committed:
[i/176] removing "•" and "&bull;"

senky committed:
[i/179] replacing empty() with equality check

senky committed:
[i/180] Contact info category in profile should be displayed only if it contains
some content

Old commit messages:
[i/180] adding condition for Contact Info container

[i/180] removing extra line

Kamahl19 committed:
update

Kamahl19 committed:
[i/187] update watermark

Kamahl19 committed:
[i/188] update expander plugin

Kamahl19 committed:
renamed js file

senky committed:
[i/199] removing unnecessary part of logout link in menu block

senky committed:
Merge pull request #169 from Senky/i/169

Smilies in chatbox are not displayed

senky committed:
Merge pull request #176 from Senky/i/176

When deletting status, • should not be displayed

senky committed:
Merge pull request #186 from Kamahl19/i/186

Update jquery.detectmobilebrowser

senky committed:
Merge pull request #180 from Senky/i/180

Contact info category in profile should be displayed only if it contains some content

senky committed:
Merge pull request #179 from Senky/i/179

It isn't possible to create friends group called "0"

senky committed:
Merge pull request #175 from Senky/i/175

Submitting empty status results in not collapsed textarea

senky committed:
Merge pull request #174 from Senky/i/174

Suggestions user blocks does not have float: left

senky committed:
Merge pull request #171 from Senky/i/171

Reporting user is not properly styled for high resolution screens

senky committed:
Merge pull request #187 from Kamahl19/i/187

Update jquery.watermark

senky committed:
Merge pull request #188 from Kamahl19/i/188

Update jquery.expander

senky committed:
Merge pull request #172 from Senky/i/172

People you may know block displays people of non-confirmed friends

senky committed:
Merge pull request #173 from Senky/i/173

Mutual friends are not displayed because of sql error

senky committed:
Merge pull request #199 from Senky/i/199

Remove unnecessary parametres from logout link in menu block

Kamahl19 committed:
Merge branch 'i/76' of https://github.com/Kamahl19/phpBB-Social-Network into develop

Kamahl19 committed:
[i/76] fixed lang

senky commented:
Do this:
1. `git reset --hard 91eae3a`
2. `git fetch upstream`
3. `git rebase upstream/develop`
4. resolve the merging conflicts
5. https://github.com/Kamahl19/phpBB-Social-Network/commit/91eae3a199ab24f0f2426f5bc8c1bf2c9536973f#root-language-en-mods-socialnet_acp-php-P5
6. git add and git commit
7. `git push -f origin i/76`

Kamahl19 commented:
thanks, how can i resolve merge conflict? do you an idea why this happened ?

senky commented:
`git mergetool`

Conflicts happen, when you edit lines, which have been modified by another PR, which was closed earlier than this one.
